src/ngraph/op/non_max_suppression.hpp

```diff
@@ -92,8 +94,95 @@ namespace ngraph
             private:
                 int64_t max_boxes_output_from_input() const;
             };
-        }
-    }
+        } // namespace v1
+
+        namespace v3
+        {
+            /// \brief NonMaxSuppression operation
+            ///
+            class NGRAPH_API NonMaxSuppression : public Op
+            {
+            public:
+                enum class BoxEncodingType
+                {
+                    CORNER,
+                    CENTER
+                };
+
+                static constexpr NodeTypeInfo type_info{"NonMaxSuppression", 1};
```

Comment:
There will be trouble in River City if you don't change the version to 3.

Reply:
Missed that one.  Will correct.